Utils/Elasticsearch/mapping/tasks.mapping

```diff
@@ -236,6 +236,9 @@
         },
         "n_files_to_be_used": {
           "type": "long"
+        },
+        "update_required": {  /* service field to mark  documents that contain incomplete */
```

Comment:
Extra space: `mark__documents`.

Reply:
[Fixed](https://github.com/PanDAWMS/dkb/compare/4afb4dd9b8c311b30a59fab1b0dfe7e221498f37..a2e2cc48dd6f04744d458af5dafede69b67ce57d), thank you!